apps/ledger-live-mobile/src/components/DeviceAction/index.tsx

```diff
@@ -322,23 +409,13 @@ export function DeviceActionDefaultRendering<R, H, P>({
     // NB Until we find a better way, remap the error if it's 6d06 and we haven't fallen
     // into another handled case.
     if (
-      error instanceof DeviceNotOnboarded ||
-      (error instanceof TransportStatusError &&
-        error.message.includes("0x6d06"))
+      device &&
+      (error instanceof DeviceNotOnboarded ||
+        ((error as unknown) instanceof TransportStatusError &&
+          (error as Error).message.includes("0x6d06")))
     ) {
       return renderDeviceNotOnboarded({ t, device, navigation });
     }
-
-    return renderError({
-      t,
-      navigation,
-      error,
-      managerAppName:
-        error.name === "UpdateYourApp" ? error.managerAppName : undefined,
-      onRetry,
-      colors,
-      theme,
-    });
```

Comment:
Ok i was panicking because some device action were not working 😱

Actually, it's because some errors were not handled anymore

This `renderError` should not have been deleted ;) it needs to be put back. I can do the commit if you want ? Also there is the problem of the `error` type, where we have the `managerAppName` prop that is not defined on the current `error` type 😅

Reply:
Thx !